<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorWidget</class>
 <widget class="QWidget" name="ColorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>44</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>226</width>
    <height>44</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>226</width>
    <height>44</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_1">
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_4">
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton_5">
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pushButton_6">
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="pushButton_7">
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="pushButton_8">
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
